apps/server/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "server",
```

Comment:
Used scoped packages here, and in the rest of the packages and apps.

Reply:
scoped them